inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/gather_elements_ref.cl

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "include/data_types.cl"
+#include "include/fetch_data.cl"
+
+#define GET_UPDATES_INDEX(prefix, idx_order) CAT(prefix, _GET_INDEX)(idx_order)
```

Comment:
Seems that this def is coming from scatter update series. Please rename this according to gather ....

Reply:
I'll change it to GET_OUTPUT_INDEX.